test/packages/other/synthetic_mode/data_stream/synthetic/sample_event.json

```diff
@@ -0,0 +1,94 @@
+{
+    "@timestamp": "2023-06-07T08:30:02.333Z",
+    "agent": {
+        "ephemeral_id": "296ade19-2a9f-47f6-95f9-dcfa40559fd0",
+        "id": "86e09a11-1c59-4cb9-9eda-2b025a06143e",
+        "name": "docker-fleet-agent",
+        "type": "metricbeat",
+        "version": "8.7.1"
+    },
+    "data_stream": {
+        "dataset": "synthetic_mode.synthetic",
+        "namespace": "ep",
+        "type": "metrics"
+    },
+    "ecs": {
+        "version": "8.7.0"
+    },
+    "elastic_agent": {
+        "id": [
+            "86e09a11-1c59-4cb9-9eda-2b025a06143e"
+        ],
+        "snapshot": [
+            false
+        ],
+        "version": [
+            "8.7.1"
+        ]
```

Comment:
Umm, why are these values left as arrays? I don't think they have any normalization.

Is it because they don't have a definition (from [here](https://github.com/elastic/elastic-package/pull/1283/files#diff-36d325d58b4ea8957abc63576d2e04d30ec8e58435ba06d442cb7923af063ad3R392))? Maybe for these cases it is better then that we use the single value when the array has only one element, and the array if there are more elements.

Reply:
Exactly, as those fields do not have any definition they are kept as they are in the document (`fields` key)